### PR TITLE
Consolidating notes

### PR DESCRIPTION
**Pull request by ernesmb**
opening PR with first addings

cc/ @jsanz

### CODE REVIEW

File: 01-builder-visualization/README.md [hunk 1]
```diff
@@ -19,3 +19,8 @@ CARTO BUILDER for Data Visualization
 * Exercise: [category-bubbles](exercises/category-bubble.md)
```
Comment (jsanz) [inline 01-builder-visualization/README.md]: I don't like how they render I would use two lists of links with single headers, simpler and easy to read like you did on the day 2

Comment (ernesmb) [inline 01-builder-visualization/README.md]: agree, I wasn't sure about which format was better, so I used both and waited for your feedback :)

**Comment by jsanz** [comment]
Awesome Ernesto, great review!!

:+1:

**Comment by ernesmb** [comment]
thanks! :) shipping it then!

**Pull request merged**